app/assets/javascripts/discourse/app/components/bread-crumbs.js

```diff
@@ -44,7 +31,7 @@ export default Component.extend({
         options,
         isSubcategory: !!parentCategory,
         noSubcategories: !category && noSubcategories,
-        hasOptions: options.length !== 0,
+        hasOptions: !parentCategory || parentCategory.has_children,
```

Comment:
I'm not sure I understand the need for`parentCategory.has_children` here because the fact that the `parentCategory` exists mean it has child categories. Otherwise, `parentCategory` would be blank.